hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -78,6 +78,7 @@ public static EthTxData populateEthTxData(byte[] data) {
             return switch (rlpItem.asByte()) {
                 case 1 -> populateEip2390EthTxData(decoder.next(), data);
                 case 2 -> populateEip1559EthTxData(decoder.next(), data);
+                case 3 -> null; // We don't currently support Cancun "blob" transactions
```

Comment:
There are callers of `populateEthTxData` that do not check for a null return value.  Example `MiscUtils` lines 778-781, `HandleWorkflow` lines 829-831, `ThrottleAccumulator` and other places.  Some of these are for mono service and some in modularized code.  I think the saving grace is a check in `IngestChecker` (at least for modularized) service that checks for a null and does not let the transaction proceed.  It seems like we might have a problem with NullPointerExceptions if a transaction ever escaped this check.  
Would it make sense to do null checks everywhere `populateEthTxData` is called or are we safe because of that check in `IngestChecker`?

Reply:
Hmm, good investigating, TY! I'll look more closely (as I had _assumed_ the `IngestChecker` was sufficient but didn't go further).

Reply:
OK, I fixed `MiscUtils`, `ThrottleAccumulator` has both uses guarded, `SignedTxnAccessor` is guarded, `HandleWorkflow` got the same fix as `MiscUtils`, and `IngestChecker` checks for nulls there (and throws).

All other uses are in unit tests where the test will end up failing (one way or another) if it gets a null.

Reply:
Do we have any EETs that submit `EthereumTransaction` with invalid bytes? Seems likely, but not sure.